include/vkgcDefs.h

```diff
@@ -625,6 +625,7 @@ struct GraphicsPipelineBuildInfo {
 
   NggState nggState;       ///< NGG state used for tuning and debugging
   PipelineOptions options; ///< Per pipeline tuning/debugging options
+  bool unlinked;           ///< True to build an "unlinked" half-pipeline ELF
```

Comment:
It seems this flag is used in standalone compiler. Maybe add it in comments: (for standalone compiler use)

Reply:
Once we have shader compilation and linking working, it will be used from the driver too.